### PR TITLE
Ref/41677

### PR DESCRIPTION
**Pull request by boddy-devbranch**

### CODE REVIEW

File: modules/living_spaces_event/living_spaces_event.module [hunk 1]
```diff
@@ -319,6 +320,38 @@ function living_spaces_event_living_spaces_event_presave(LivingSpaceEventInterfa
   }
 }
 
+/**
+ * Implements hook_ENTITY_TYPE_insert().
+ */
+function living_spaces_event_living_spaces_event_invite_insert(LivingSpaceEventInviteInterface $invite) {
+  // Ref #41677 - create notification and email to invited user to the event.
```
Comment (victor-aston) [inline modules/living_spaces_event/living_spaces_event.module]: It's a contrib module. So, no need to add any refs to redmine tasks.

File: modules/living_spaces_event/src/Form/LivingSpaceEventInviteUsersForm.php [hunk 2]
```diff
@@ -3,7 +3,6 @@
 namespace Drupal\living_spaces_event\Form;
 
 use Drupal\Core\Entity\Element\EntityAutocomplete;
-use Drupal\Core\Entity\EntityTypeManagerInterface;
```
Comment (victor-aston) [inline modules/living_spaces_event/src/Form/LivingSpaceEventInviteUsersForm.php:6]: no need to remove it.

File: modules/living_spaces_event/src/Form/LivingSpaceEventInviteUsersForm.php [hunk 3]
```diff
@@ -93,7 +98,14 @@ public function submitForm(array &$form, FormStateInterface $form_state) {
...
           else {
-            $this->messenger()->addWarning($this->t('User doesn\'t have a membership in this space.'));
+            $member = $this->entityTypeManager->getStorage('user')->load($match);
+            $message = $this->t('<strong>@user</strong> could not be invited to this @event event, because they are missing a membership in this space.', [
+              '@user' => $member->getDisplayName(),
+              '@event' => $event->toLink($event->label())->toString(),
+            ]);
+            $this->logger('Space event')->notice($message);
+            $this->messenger()->addWarning($message);
+
```
Comment (victor-aston) [inline modules/living_spaces_event/src/Form/LivingSpaceEventInviteUsersForm.php]: empty line

File: modules/living_spaces_event/src/Form/LivingSpaceEventInviteUsersForm.php [hunk 4]
```diff
@@ -22,22 +21,28 @@ class LivingSpaceEventInviteUsersForm extends FormBase {
...
+   * The mail manager.
    *
-   * @param \Drupal\Core\Entity\EntityTypeManagerInterface $entity_type_manager
-   *   Provides an interface for entity type managers.
+   * @var \Drupal\Core\Mail\MailManagerInterface
    */
-  public function __construct(EntityTypeManagerInterface $entity_type_manager) {
-    $this->entityTypeManager = $entity_type_manager;
-  }
+  protected $mailManager;
```
Comment (victor-aston) [inline modules/living_spaces_event/src/Form/LivingSpaceEventInviteUsersForm.php]: we don't use mail manager and config factory for the form anymore. you can remove it.
in order words, you can leave changes related to the warning message. that's all we need to update.

File: modules/living_spaces_event/src/Form/LivingSpaceEventInviteUsersForm.php (hunk 2 above)
Comment (boddy-devbranch) [reply modules/living_spaces_event/src/Form/LivingSpaceEventInviteUsersForm.php:6]: It was used in the construct but it is not used anymore

**Review by victor-aston — APPROVED** [review]

**Review by andrew-sereda — APPROVED** [review]

**Pull request merged**